sdk/storage/azure-storage-extensions/pyproject.toml

```diff
@@ -14,7 +14,7 @@ bandit = false
 suppressed_skip_warnings = ["*"]
 
 [tool.cibuildwheel]
-build = ["cp38*", "cp39*", "cp310*", "cp311*", "cp312*", "pp38*", "pp39*", "pp310*"]
```

Comment:
Did we intentionally remove `312`?

Reply:
Yep. We removed everything except our current min supported version since the new wheels produced will be forwards compatible with all versions going forward.